content/docs/tooling/transactions/native-send.mdx

```diff
@@ -0,0 +1,109 @@
+---
+title: Send AVAX on C/P-Chain
+description: Learn how to execute a native transfer on the C or P-Chain using the Avalanche CLI.
+---
+
+# Prerequisites
+
+- Install the [Avalanche CLI](https://docs.avax.network/tooling/guides/get-avalanche-cli).
+- Use the CLI to [create a key](https://docs.avax.network/tooling/avalanche-cli#key-create).
+- Fund the key with AVAX. You can use the [faucet](https://test.core.app/tools/testnet-faucet/?subnet=c&token=c) with coupon code `devrel-avax-0112` to get testnet AVAX. 
```

Comment:
is it ok to give out this coupon? I thought Ash said something about this yesterday

Reply:
this is the general coupon, unrelated to Ash's convo. If this coupon is abused we can terminate it's use pretty easily. I wouldn't want anyone to follow this tutorial and be completely unable to get testnet tokens.